src/CoreFoundation/CFArray.cs

```diff
@@ -35,6 +35,8 @@
 using ObjCRuntime;
 
 using CFIndex = System.nint;
+using CFArrayRef = System.IntPtr;
+using CFAllocatorRef = System.IntPtr;
```

Comment:
Why do we name these like this and not just use `IntPtr`directly for the parmeter types in `Clone()`? There's other bindings that just use IntPtr directly.

Reply:
It is easier to read. I would have changed all, but would have increase the diff. It does not change the output result of the binding.

Reply:
Haha nice question, but yeah just for readability.

![giphy](https://user-images.githubusercontent.com/204671/68628545-e7eb4e80-04ae-11ea-92b2-755da6e51d5d.gif)